StortSpelProjekt/Game/src/Gamefiles/UpgradeManager.h

```diff
@@ -17,6 +17,7 @@
 enum E_RangeName
 {
 	UPGRADE_RANGE_TEST = 1,
+	UPGRADE_MELEE_TEST = 2,
```

Comment:
Why is melee test in range name?

Reply:
There was a small change in how this enum is used and the name change was forgotten, good catch.